app/ProjectAanvraag/ProjectAanvraagServiceProvider.php

```diff
@@ -25,7 +25,7 @@ public function register(): void
             return new ProjectAanvraagClient(
                 new Client(
                     [
-                        'base_uri' => config('project_aanvraag.base_uri'),
+                        'base_uri' => config('project_aanvraag.base_uri.live'),
```

Comment:
This becomes more tricky now, this is also dependent on the integration that gets created. So I think the HTTP client needs to be created inside the `createWidget` method depending on the Integration status.

Reply:
Maybe the easiest approach is to extend the `CreateWidgetRequest` with the `IntegrationStatus`